src/common/validators/email.validator.ts

```diff
@@ -12,8 +12,7 @@ export const IsEmail = (
       constraints: [options],
       validator: {
         validate: (value, args) =>
-          value !== null ||
```

Comment:
Oh derp. We said anything not null is allowed. I'd like to continue checking and allowing null/undefined here as it's possible that those get passed in here and we don't want to enforce "required" only that the format if given is an email.
```suggestion
          value == null ||
          (typeof value === 'string' && isEmail(value, args?.constraints[0])),
```